PocketKit/Sources/PocketKit/Home/HomeCarouselItemCell.swift

```diff
@@ -165,8 +164,12 @@ extension HomeCarouselItemCell {
 
         if model.attributedTimeToRead.string.isEmpty {
             timeToReadLabel.isHidden = true
+            subtitleStack.setCustomSpacing(0, after: timeToReadLabel)
+            subtitleStack.setCustomSpacing(Constants.layoutMargins.bottom, after: domainLabel)
         } else {
             timeToReadLabel.isHidden = false
+            subtitleStack.setCustomSpacing(Constants.stackSpacing, after: domainLabel)
+            subtitleStack.setCustomSpacing(Constants.layoutMargins.bottom, after: timeToReadLabel)
```

Comment:
need to make sure that the bottom most element of the stack was 16.0 away from the bottom of the stack to match the buttons, open to other suggestions though!